iohk-monitoring/src/Cardano/BM/Backend/ExternalAbstraction.lhs

```diff
@@ -63,9 +62,9 @@ instance Pipe NoPipe where
     write   = \_ _ -> pure ()
     getLine = \_ -> pure ""
 
+#ifndef mingw32_HOST_OS
 instance Pipe UnixNamedPipe where
     data PipeHandler UnixNamedPipe = P Handle
```

Comment:
Do you need type families for this?

Reply:
maybe not. at some point a Windows named pipe implementation would also be necessary.